WcaOnRails/app/controllers/api/v0/competitions_controller.rb

```diff
@@ -1,11 +1,21 @@
 # frozen_string_literal: true
 class Api::V0::CompetitionsController < Api::V0::ApiController
   def show
-    competition = Competition.where(id: params[:id], showAtAll: true).first
-    unless competition
+    competition = Competition.visible.find_by(id: params[:id])
+    if competition
+	    render json: competition
+    else
       render json: { error: "Competition with id #{params[:id]} not found" }, status: 404
-      return
     end
-    render json: competition
+  end
+
+  def results
+  	competition = Competition.visible.find_by(id: params[:id])
+  	if competition
+  		results = Result.search_by_competition(competition.id, params: params)
+  		render json: Result.search_by_competition(competition.id, params: params)
+  	else
+  		render json: { error: "Competition with id #{params[:id]} not found" }, status: 404
+  	end
   end
 end
```

Comment:
@coder13, our linter should be checking stuff like this. I don't want our code reviews to get littered with style nitpicks. Can you check if our linter actually does detect incorrect indentation? If not, could you investigate a little and create an issue for it?